docs/configuration/vpn/sstp.rst

```diff
@@ -471,6 +471,10 @@ Global Advanced options
 
   Windows Internet Name Service (WINS) servers propagated to client
 
+.. cfgcmd:: set vpn sstp host-name <hostname>
+
+  Only allow connection to specified host with the same TLS SNI
```

Comment:
Can we use the sentence from MAN page: `If this option is given, only sstp connection to specified host and with the same TLS SNI will be allowed.`

Reply:
Changed